dirty_cat/test/test_minhash_encoder.py

```diff
@@ -1,5 +1,7 @@
 import numpy as np
 import time
+import pytest
+from sklearn.utils._testing import assert_raise_message
```

Comment:
I worry that we are importing from a private submodule of sklearn, and this may break without warning.

I think that I would prefer that we use the pattern listed here:
https://docs.pytest.org/en/stable/assert.html#assertions-about-expected-exceptions